reconcile/rhidp/ocm_oidc_idp/base.py

```diff
@@ -55,7 +57,13 @@ def run(
             desired_state = fetch_desired_state(
                 secret_reader, clusters, vault_input_path
             )
-            act(dry_run, ocm_map, current_state, desired_state)
+            act(
+                dry_run,
+                ocm_map,
+                current_state,
+                desired_state,
+                managed_idps=managed_idps if managed_idps else [],
```

Comment:
nit. feel free to ignore

```suggestion
                managed_idps=managed_idps or [],
```

Reply:
I really wish python has `None` aware operators https://peps.python.org/pep-0505/, use `or` to set default can fail when it's optional boolean.